doc/classes/Curve2D.xml

```diff
@@ -177,5 +177,8 @@
 		<member name="point_count" type="int" setter="set_point_count" getter="get_point_count" default="0">
 			The number of points describing the curve.
 		</member>
+		<member name="preview_resolution" type="int" setter="set_preview_resolution" getter="get_preview_resolution" default="32">
+			The number of intervals between two spline points. The larger the resolution, the more preview items are drawn. Lowering this value can help with responsiveness when editing the curve.
+		</member>
```

Comment:
I'd prefix the property name with `debug` to make it clear that this property only affects debug drawing. The description should make note of this as well.